.github/workflows/int_test.yml

```diff
@@ -94,7 +107,7 @@ jobs:
       matrix:
         feature: [custom-input, default, insecure-listener, syncrules, redeploy, trustbundle]
         runtime-args:
-          - ${{ inputs.runtime-init-args != '' || inputs.runtime-create-args != '' }}
+          - ${{ inputs.runtime-create-args != '' || inputs.runtime-init-args != '' }}
```

Comment:
Can't use the env vars here ?

Reply:
done - also tried to do this for the runtime args too (discussed that the matrix still needs the inputs)

Reply:
Hmm @c-ryan-k do you have a suggestion?